support-frontend/assets/components/content/content.scss

```diff
@@ -101,6 +101,62 @@
   padding: ($gu-v-spacing/2) ($gu-h-spacing / 2) 0;
 }
 
+.component-content__container--full-width {
+  @include mq($from: wide) {
+    width: 100%;
+    max-width: 100%;
+    display: flex;
+    padding-left: 70px;
+  } 
+  
+  @include mq($from: 1395px) {
+    justify-content: center;
+    padding-left: 0;
+    margin-left: -35px;
+  }
+
+  .component-content__content--centered {
+    max-width: gu-span(4);
+    padding: $gu-v-spacing / 2 0 $gu-v-spacing * 3 10px;
+  
+    @include mq($from: mobileLandscape) {
+      padding-left: 10px;
+    }
+  
+    @include mq($from: phablet) {
+      max-width: gu-span(6);
+      padding-left: 22px;
+    }
+
+    @include mq($from: tablet) {
+      padding-left: 45px;
+    }
+  
+    @include mq($from: desktop) {
+      max-width: gu-span(8);
+      padding-left: 70px;
+    }
+  
+    @include mq($from: wide) {
+      padding-left: 0px;
+      max-width: 1170px;
+      margin-left: -10px;
+    }
+  
+    .hope-is-power--centered {
```

Comment:
This seems a very specific class to have in this generic component

Reply:
Yes, it is. The name is there to differentiate this campaign from the one called 'the-moment'. This is a very campaign-driven page, so in this context the specificity seems justifiable to me.